RELEASE_PLAN.md

```diff
@@ -0,0 +1,58 @@
+## Release Plan
+
+Starting with version `v2.4`, the Nuxt core team will adhere to a formalized release plan (as good as possible).
+Also, an end of life for older major versions is defined with this document
+
+### Major versions (3.x.y, 4.x.y, ...)
+
+We aim for a major release once every **6 months**. This depends on a few factors though:
+
+* Major version releases of critical dependencies like Vue, Webpack or Babel
+* Number of breaking changes "in the queue"
+
+Our goal is to provide a **migration guide** for each major version as well as escape hatches so existing code
+won't "just break".
+
+### Minor versions
+
+The team agree on a release cycle of (roughly) **4 weeks**. 
+
+Three of the for weeks will be used for actual **feature implementations** while the last week will be used for 
+**testing, fixing bugs and thorough audits**.
+
+That also means a *feature freeze* for the next minor version after these three weeks.
+Features that aren't ready will be moved to the next cycle. We try to avoid to "wait" for features 
+(for a longer time) to keep releases lean, concise, predictable and digestible.
+
+### Patch releases
+
+The last patch releases were mostly *bundled* fixes or single *hotfixes*. 
+In the future we will avoid this practice and try to release each fix **as soon as possible** after the actual PR/commit.
+People won't have to switch to `nuxt-edge` for bugfixes then.
+
+Fixes can or will include:
+
+* Updates of dependencies (for various reasons, like a "faulty/buggy" dependency or an newer versions that works better with the Nuxt.js code)
+* Fixes for our code
+
+Bugfixes for upcoming features won't be ported of course.
+
+### Edge releases
+
+We've experimented with `nuxt-edge` releases in the last time and decided to do **nightly releases** for now instead of 
+releasing a version after *each commit*.
+
+## End Of Life
+
+We agreed to set an **End Of Life** for older major Nuxt.js versions. 
+Previous releases will receive security updates and bugfixes *until two weeks after the second next major release*. 
```

Comment:
I suggest adding: except critical security fixes which will be backported.

Reply:
Would you guarantee critical fixes for Nuxt 1.x though we are on `5.x` for example? :thinking:

Reply:
NPM packages are not marked as obsolete. So logically we have to do this if get security report emails for them.

Reply:
Is it possible to deprecate/archive specific *version* of packages?
If so, I'd vote for that then.
People who didn't upgrade in that time likely wouldn't update the package as well (from my experience).

Reply:
Yes, it is possible :) But we should really talk more about it as Nuxt has active users on 0.x/1.x. Deprecating those versions without proper migration docs (and the fact that we didn't provide any release plan) is a little confusing. Meanwhile, I suggest to just mention the fact that non-obsolete versions _may occentially_ receive critical fixes.